rabbitmq-operator-model/src/main/java/com/indeed/operators/rabbitmq/model/crd/rabbitmq/PolicyDefinitionSpec.java

```diff
@@ -0,0 +1,163 @@
+package com.indeed.operators.rabbitmq.model.crd.rabbitmq;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.indeed.rabbitmq.admin.pojo.Definition;
+
+public class PolicyDefinitionSpec {
+
+    private final String alternateExchange;
+    private final String deadLetterExchange;
+    private final String deadLetterRoutingKey;
+    private final Long expires;
+    private final String haMode;
+    private final Long haParams;
+    private final String haPromoteOnShutdown;
+    private final Long haSyncBatchSize;
+    private final String haSyncMode;
+    private final Long maxLength;
+    private final Long maxLengthBytes;
+    private final Long messageTtl;
+    private final String queueMasterLocator;
+
+    public PolicyDefinitionSpec(
```

Comment:
@amccormick-indeed what are your thoughts about having these "spec" classes? Unfortunately this one is almost a direct copy of the rabbitmq-admin Definition class, but I feel like having the separation between the two realms (k8s and rabbit) is a good thing despite the duplicated code...

Reply:
I agree with your thinking.